src/main/webapp/meal.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Meal page</title>
+    <link rel="stylesheet" href="meal-page-style.css"></link>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyA0FR7vWT5PKPh-Sq95zAjcWXBFXRYl5xI"></script>
```

Comment:
Please, follow [Google's tutorial](https://developers.google.com/maps/documentation/javascript/get-api-key) to restrict maps API key. 
Without the restriction, anyone that sees the key can copy it and use it for their own requests, and it's your account which will still be charged. Because of how web pages work, we *have* to deliver this key to user's computer (as opposed to keeping it to ourselves and never revealing it), but we should restrict access so that the key can only be used for your particular web page. This will need extra preparation - registering the HTTP referalls (e.g. on behalf of what web page this key can be used) needs the page's URL, most often the domain name. We might need to get something for you for that.

Reply:
I have such discussion with Tomasz before in one of PR. So I already restricted my key to app engine machine localhost address only (according to tutorial you mentioned).